composer.lock

```diff
@@ -648,5 +652,5 @@
     "platform-overrides": {
         "php": "7.0"
     },
-    "plugin-api-version": "1.1.0"
+    "plugin-api-version": "2.0.0"
```

Comment:
I used Composer 2 to update the lock, but it seems like it made this change. What is the best version to use?